utils/compare_two_images.py

```diff
@@ -55,18 +55,26 @@
     raise Exception("Importing skimage.data failed")
 
 try:
-    import skimage as ski
-    version = ski.__version__
+    import skimage
+    version = skimage.__version__
     print("skimage version: ",version)
     v = version.split('.')
+    # determines skimage version function
+    new_version = 0
```

Comment:
well, what if they decide in version 14 that a third name might be better... then it's not black or white anymore